libs/stats/odc/stats/_plugins.py

```diff
@@ -14,6 +15,14 @@ def _new(plugin_class, *args, **kwargs) -> StatsPluginInterface:
 
 def resolve(name: str) -> PluginFactory:
     maker = _plugins.get(name)
+    if maker is None:
+        maker = pydoc.locate(name)
+        if maker is not None:
+            mro = getattr(maker, "__mro__", tuple())
```

Comment:
What's wrong with `issubclass(maker, StatsPluginInterface)`?

Reply:
nothing, could not google it, is all